recipes/hpyx/recipe.yaml

```diff
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: hpyx
+  version: "2025.8.28"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: ce582d60d0ad50256fc8f6666d5c30be9f15af8936a8711c5f2f03404caf2958
+
+build:
+  script:
+    - if: osx
+      then: |
+        export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+    - python -m pip install . --no-deps --ignore-installed -vvv
+  number: 0
+  python:
+    # Option to specify whether a package is version independent (aka ABI3)
+    version_independent: true  # defaults to fals
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compiler names include 'c', 'cxx' and 'fortran', among others.
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+  host:
+    - python 3.13.* *_cp313t
```

Comment:
@isuruf, I never reviewed a recipe that requires cp313t, do you mind taking a look before we merge this one?

Reply:
Hi @ocefpaf and @isuruf any updates on this? Thank you both for your help!

Reply:
This is fine in staged-recipes, but we should apply the migrators in the feedstock.